test/scripts/build-spec.sh

```diff
@@ -7,7 +7,7 @@ set -e
 cd $(dirname $0)/..
 
 mkdir -p specs
-../target/release/container-chain-simple-node build-spec --disable-default-bootnode --add-bootnode "/ip4/127.0.0.1/tcp/33049/ws/p2p/12D3KooWHVMhQDHBpj9vQmssgyfspYecgV6e3hH1dQVDUkUbCYC9" --parachain-id 2000 --raw > specs/template-container-2000.json
-../target/release/container-chain-frontier-node build-spec --disable-default-bootnode --add-bootnode "/ip4/127.0.0.1/tcp/33050/ws/p2p/12D3KooWFGaw1rxB6MSuN3ucuBm7hMq5pBFJbEoqTyth4cG483Cc" --parachain-id 2001 --raw > specs/template-container-2001.json
+../target/release/container-chain-simple-node build-spec --disable-default-bootnode --parachain-id 2000 --raw > specs/template-container-2000.json
+../target/release/container-chain-frontier-node build-spec --disable-default-bootnode --parachain-id 2001 --raw > specs/template-container-2001.json
```

Comment:
Undo, it turns out those bootnodes are not needed, but this shouldn't be in this PR